### PR TITLE
Add members' and projects' page

### PR DESCRIPTION
**Pull request by bbahd30**
Created a new branch along with commits from the previous forked repo. @sweety-sg kindly review the changes.

### CODE REVIEW

File: src/components/alumni/alumni-page.js
Comment (nikkuAg) [inline src/components/alumni/alumni-page.js:1]: Either use class-based or functional based everywhere

File: src/components/app.js [hunk 1]
```diff
@@ -151,29 +150,30 @@ class App extends Component {
       apiInfoData.projectLoaded
     ) {
       return (
-        <div styleName="blocks.container">
+        <div div styleName="blocks.container">
```
Comment (nikkuAg) [inline src/components/app.js]: extra div kyu likha?

File: src/components/blog/blog-post-card.js [hunk 2]
```diff
@@ -3,22 +3,23 @@ import { Card, Container, Grid } from 'semantic-ui-react'
...
-            <div styleName='styles.blogHead'>
-                <div className="author">{info.author}</div>
-                <div className="time">5 min read</div>
+        <Grid.Column styleName="styles.blogGrid">
+            <div styleName="styles.blogInfo">
+                <div styleName="styles.blog" style={backgroundImageStyle(info.thumbnail)}
+                    onClick={() => window.open(info["link"])} />
+                <div styleName='styles.blogHead'>
+                    <div className="author">{info.author}</div>
+                    <div className="time">5 min read</div>
```
Comment (nikkuAg) [inline src/components/blog/blog-post-card.js]: Time hard code krna ha ya baad mai change kroge

File: src/components/header/app-header-browser-grid.js [hunk 3]
```diff
@@ -2,149 +2,88 @@ import React, { Component } from 'react'
...
-            >
-              <button
-                name="team"
-                styleName={`browserStyles.${this.handleNavStyle()}`}
-              >
-                Alumni
-              </button>
+          </div>
+          <div name="culture" styleName="browserStyles.btn">
+            <NavLink /* to={`${urlAppMember()}`} */ to={`${urlAppBase()}`} activeStyle={ACTIVE_STYLE.culture} >
```
Comment (nikkuAg) [inline src/components/header/app-header-browser-grid.js]: Comment hta de

File: src/components/main/sections/blog/blog-section.js
Comment (nikkuAg) [inline src/components/main/sections/blog/blog-section.js:1]: Why class-based component, if all other are functional based

File: src/components/main/sections/info/info-section.js [hunk 4]
```diff
@@ -15,115 +17,112 @@ const InfoSection = ({ location, contact, social, about }) => {
...
             </div>
-          </Grid.Column>
-        </Grid>
-        <hr />
-      </Container>
-    </div>
+          </div>
+        </Grid.Column>
+      </Grid>
+      {/* <hr /> */}
```
Comment (nikkuAg) [inline src/components/main/sections/info/info-section.js]: Remove comment

File: src/components/main/sections/project/project-grid-layout.js
Comment (nikkuAg) [inline src/components/main/sections/project/project-grid-layout.js:1]: Either class based or functional based

File: src/components/member-individual-view.js [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+  handleItemClick = (e, { name }) => {
+    this.setState({ activeTab: name })
+  }
+
+  renderContent = () => {
+    if (this.state.projectTab) {
+      return (
+        <>
+          {this.state.memberProjects.map((info, index) => (
+            <ProjectDetail /* key={index} */ info={info} key={info.slug} />
```
Comment (nikkuAg) [inline src/components/member-individual-view.js]: remove comment

File: src/components/member-individual-view.js [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+                    {!formalTheme && (
+                      <div styleName="styles.personality">
+                        <div styleName="styles.avatar"></div>
+                        <div styleName="styles.personalityType">
+                          <span>{this.state.memberDetails.personalityType}</span>
+                        </div>
+                      </div>
+                    )}
+
+                    {/* <div styleName="styles.role-designation">
```
Comment (nikkuAg) [inline src/components/member-individual-view.js]: Remove comment

File: src/components/team/member-card.js [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+                                            info.maintainer.designation === designation.value && (
+                                                <div key={designation.displayName} styleName="styles.designation">
+                                                    {designation.displayName}
+                                                </div>))}
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </Link>
+                <Link to={"../" + member + "/" + info.informalHandle}>
```
Comment (nikkuAg) [inline src/components/team/member-card.js:52]: Use the format option `${member}`

File: src/components/team/member-page.js [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+            return (
+                <>
+                    {this.props.apiTeamData.data.map(info => (
+                        <MemberCard
+                            info={info}
+                            key={info.handle}
+                            roleOptions={teamRoleOptions}
+                            designationOptions={teamDesignationOptions}
+                            member="team"
+                        // linkOptions={linkOptions}
```
Comment (nikkuAg) [inline src/components/team/member-page.js]: Remove comment

File: src/css/blog/blog-post-card.css [hunk 9]
```diff
@@ -1,15 +1,17 @@
 .blog
 {
-    width: fit-content;
+    /* width: fit-content; */
```
Comment (nikkuAg) [inline src/css/blog/blog-post-card.css]: remove comment

File: src/css/blog/blog-section.css [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+
+.category-btn{
+    font-family: 'Open Sans';
+    cursor: pointer;
+    padding-right: 0.9rem;
+    font-weight: 600;
+    font-size: 1.1rem;
+}
+
+.verticalCategory > *{
```
Comment (nikkuAg) [inline src/css/blog/blog-section.css]: why > *

File: src/css/blog/blog-section.css [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+    margin-bottom: 7.6rem !important;
+}
+
+.verticalCategory > * > div{
+    padding-right: 3.4rem;
+}
+.verticalCategory > * > div:first-child{
```
Comment (nikkuAg) [inline src/css/blog/blog-section.css]: provide a new class

File: src/css/blog/blog-section.css [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+    .horizontalCategory > * > div, .verticalCategory > * > div{
+    padding-right: 1.5rem !important;
+  }
+}
+
+@media (max-width: 590px) {
+    .horizontalCategory > * > div, .verticalCategory > * > div{
+    padding-right: 0 !important;
+  }
+}
```
Comment (nikkuAg) [inline src/css/blog/blog-section.css]: Add a new line at end

File: src/css/footer/app-footer.css [hunk 13]
```diff
@@ -21,4 +17,35 @@
...
+  .grpInfo{
+    margin: 0 3.4rem 0 3.8rem;
+  }
+}
+
+@media (max-width: 768px){
+    .grpInfo{
+        padding-top: 0; 
+    }
+}
```
Comment (nikkuAg) [inline src/css/footer/app-footer.css]: New line at end

File: src/css/sections/info/info-section.css [hunk 14]
```diff
@@ -31,38 +44,64 @@
...
+@media (max-width: 768px) {
+  .infoSection{
+    padding: 0px 1.7rem !important;
+    border-bottom: none;
+  }
+
+  .icons2Bar{
+    border-bottom: 1px solid #F0F0F0;
+  }
+}
```
Comment (nikkuAg) [inline src/css/sections/info/info-section.css]: New line at end of file

File: src/css/sections/project/project-grid-layout.css [hunk 15]
```diff
@@ -61,4 +104,27 @@
...
+  0% {
+    opacity: 0.25;
+  }
+  50%{
+    opacity: 0.65;
+  }
+  100% {
+    opacity: 1;
+  }
 }
```
Comment (nikkuAg) [inline src/css/sections/project/project-grid-layout.css]: New line at end of file

File: src/css/utilComponents/toggleBtn.css [hunk 16]
```diff
@@ -0,0 +1,18 @@
...
+.circle{
+    position: absolute;
+    transition: 1s;
+    top: 0.22rem;
+    width: 1.6rem;
+    height: 1.6rem;
+    border-radius: 50%;
+    background-color: #FFFFFF;
+    box-shadow: 0px 1px 3px rgb(0 0 0 / 20%);
+}
```
Comment (nikkuAg) [inline src/css/utilComponents/toggleBtn.css:17]: New line at end of file

**Review by nikkuAg — CHANGES_REQUESTED** [review]
Try to use only one kind of components either Class based or Functional based

**Review by nikkuAg — CHANGES_REQUESTED** [review]
Try to use only one kind of components either Class based or Functional based

File: src/components/blog/blog-post-card.js (hunk 2 above)
Comment (bbahd30) [reply src/components/blog/blog-post-card.js]: Abhi time ki info api me nhi hai..so alag se api likhni hai iski

File: src/components/member-individual-view.js [hunk 17]
```diff
@@ -137,158 +132,130 @@ class MemberIndividualView extends Component {
...
               </div>
-              <div styleName="styles.toggle">
-                <ToggleBtn
-                  setTheme={this.props.setTheme}
-                  formalTheme={this.props.currentTheme === 'formal'}
+            </div>
+            <div styleName="styles.memberInputs">
+              <Menu text pointing secondary>
+                <Menu.Item
+                  style={{
```
Comment (sweety-sg) [inline src/components/member-individual-view.js]: can this style not be incorporated into stylesheet?

**Review by sweety-sg — APPROVED** [review]

**Review by sweety-sg — APPROVED** [review]

**Review by sweety-sg — APPROVED** [review]

**Comment by nikkuAg** [comment]
@bbahd30 
Change every component to Functional based component.

**Pull request merged**